source/includes/markdown/_09-rich-text.html.md

```diff
@@ -104,7 +104,7 @@ In addition, the following tags are supported in the rich text of project briefs
 | `<object type="application/vnd.asana.project_milestones">` | List of milestones            |
 | `<object type="application/vnd.asana.project_goals">`      | List of goals                 |
 
-**Note:** This list can expand as new features are introduced to the Asana web product. Treat rich text as you would treat arbitrary HTML, and ensure that your code doesn't break when it encounters a tag not on this list.
+**Note:** Please note that the [Project Brief](/docs/project-briefs) API is in _preview_, and is expected to change. In addition, note that the above lists can expand as new features are introduced to the Asana web product. Treat rich text as you would treat arbitrary HTML, and ensure that your code doesn't break when it encounters a tag not on this list. 
```

Comment:
Not only the project brief api, but also the new rich text features.

Reply:
Fixed 👍